docs/extend/extend-cli/cli-setting-up.md

```diff
@@ -5,43 +5,43 @@ Before you follow the development tutorials for creating a Zowe&trade; CLI plug-
 
-The `master` branch of Sample Plug-in is compatible with the `@zowe-v2-lts` version of core CLI (Zowe LTS release).
+    Clone the repository into your development folder to match the following structure:
+    ```
+    zowe-tutorial
+    └── zowe-cli-sample-plugin
+    ```
 
-The `master` branch of Sample Plug-in is also compatible with the `@latest` version of core CLI (Zowe Active Development release) at this time.
+    1. Open a terminal and enter `cd zowe-tutorial` to change directory into your `zowe-tutorial` folder.
+    2. Enter `git clone https://github.com/zowe/zowe-cli-sample-plugin` to clone the `zowe-cli-sample-plugin` repository.
+    3. Enter `cd zowe-cli-sample-plugin` to change directory into your `zowe-cli-sample-plugin` folder.
+    4. Enter `git checkout master` navigate to the `master` branch.
+    5. Enter `npm install` to install all dependencies and modules for the project.
+    6. Enter `npm run build` to create a production build.
+    
+    **[are steps 1-6 correct?]**
 
-For more information about the versioning scheme, see [Maintainer Versioning](https://github.com/zowe/zowe-cli/blob/master/docs/MaintainerVersioning.md) in the Zowe CLI repository.
+### Optional step: Run the automated tests
+**[is "the" really needed in Line 28?]**
 
-### Clone zowe-cli-sample-plugin and build from source
-Clone the repository into your development folder to match the following structure:
-```
-zowe-tutorial
-└── zowe-cli-sample-plugin
-```
-**Follow these steps:**
+We recommend running automated tests on all code changes.
 
-1. `cd` to your `zowe-tutorial` folder.
-2. `git clone https://github.com/zowe/zowe-cli-sample-plugin`
-3. `cd` to your `zowe-cli-sample-plugin` folder.
-4. `git checkout master`
-5. `npm install`
-6. `npm run build`
+To run automated tests:
 
-### (Optional) Run the automated tests
-We recommend running automated tests on all code changes. Follow these steps:
+1. Open a terminal and enter `cd __tests__/__resources__/properties` to change directory into your `__tests__/__resources__/properties` folder.
+2. Copy the content in the `example_properties.yaml` file to the `custom_properties.yaml` file. **[where does this happen? a files explorer window? in a terminal w/ a command?]**
+3. Edit the properties within `custom_properties.yaml` to contain valid system information for your site. **[what tool/app is used to edit, an ide?]**
+4. In a terminal, enter `cd zowe-cli-sample-plugin`  to change directory into your `zowe-cli-sample-plugin` folder.
+5. Enter `npm run test` to run the automated test.
 
-1. `cd` to the `__tests__/__resources__/properties` folder.
-2. Copy `example_properties.yaml` to `custom_properties.yaml`.
-3. Edit the properties within `custom_properties.yaml` to contain valid system information for your site.
-4. `cd` to your `zowe-cli-sample-plugin` folder
-5. `npm run test`
+**[are steps 1-5 correct?]**
```

Comment:
@adam-wolfe please respond

Reply:
Looking at the sample plug-in, I'm not sure where `custom_properties.yaml` is being read, unless it's handled automatically by the test framework. Need an engineer on this question.

Reply:
Yes the `custom_properties.yaml` file is automatically read by the test framework that we use for CLI system tests: https://github.com/zowe/zowe-cli/blob/6d178c2fa95d002fb8e3ad7e9ff95679243eafc1/__tests__/__packages__/cli-test-utils/src/environment/TestEnvironment.ts#L134

Reply:
@adam-wolfe @t1m0thyj -- does this mean the steps are accurate as they are?

Reply:
discussed in person